index.html

```diff
@@ -1,16 +1,95 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
     <meta charset="utf-8">
-    <title>Hello World!</title>
-    <link href="style.css">
-    <link href="styles/normalize.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>MOHAI Data</title>
+    <link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
+    <link rel="icon" href="favicon.ico" type="image/x-icon">
+    <link rel="stylesheet" href="styles/normalize.css">
+    <link rel="stylesheet" href="styles/base.css">
+    <link rel="stylesheet" href="styles/layout.css">
+    <link rel="stylesheet" href="styles/modules.css">
+    <link rel="stylesheet" href="styles/icon.css">
+    <link rel="stylesheet" href="styles/d3.css">
   </head>
-  <main>
-    <h1>HALLO WERLD</h1>
-  </main>
-  <script src="https://code.jquery.com/jquery-2.2.4.min.js"   integrity="sha256-BbhdlvQf/xTY9gja0Dq3HiwQF8LaCRTXxZKRutelT44=" crossorigin="anonymous"></script>
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/handlebars.js/4.0.5/handlebars.amd.min.js"></script>
-  <script src="app.js"></script>
-  <script src="https://cdn.rawgit.com/visionmedia/page.js/master/page.js"></script>
+  <body>
+    <div id="menuToggle" class="icon icon-menu"></div></br>
+    <nav class="icon nav_bar">
```

Comment:
Looks like some odd indentation here, which can easily be fixed with Atom's auto-indent feature. Check it out. :-)